themes/digital.gov/src/scss/new/_collection.scss

```diff
@@ -30,6 +28,17 @@
     width: 0.9em;
   }
 
+  .icon {
+    @include u-margin-top("1px");
+    @include u-margin-right(1);
+    img {
+      @include u-width(2);
+      @include u-maxw(2);
+      @include u-position("relative");
+      @include u-top("2px");
+    }
```

Comment:
Let's move away from single line property setting via mixins.

Reply:
Updated to use `units` for single line property ruels.